src/librekey/key_store_kbx.cpp

```diff
@@ -602,6 +607,12 @@ rnp_key_store_kbx_write_pgp(rnp_key_store_t *key_store, pgp_key_t *key, pgp_dest
             goto finish;
         }
     }
+    for (list_item *expiration_entry = list_front(subkey_sig_expirations); expiration_entry; expiration_entry = list_next(expiration_entry)) {
```

Comment:
Probably this will not fit line length limitation - please check with clang-format (or use shorter variable name - personally I prefer this way since one-liner code looks better).